project-icp/bee-pollinator-app
# Import Prototype

## Overview

Imports design prototype and deconstructs into very basic components. Also upgrades scripts, React, babel, Webpack, ESLint, and Style Lint. See commit messages for details.

Connects #295 

### Demo

![image](https://user-images.githubusercontent.com/1430060/47464459-3c922d80-d7b7-11e8-801f-60a32e1ac691.png)

## Testing Instructions

* Check out this branch
* Run `./scripts/beekeepers.sh install` to update all front-end dependencies
* Run `./scripts/beekeepers.sh build` to generate output
* Go to [:8000/?beekeepers](http://localhost:8000/?beekeepers) and see the prototype
